docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -318,6 +318,39 @@ sometimes resolved by additionally installing the ``apptainer-setuid`` package.
       ``slurm.slots_per_node`` to enable multiple CPUs to be used on each node. Without
       ``slurm.slots_per_node`` the job will request ``slots_per_trial`` nodes.
 
+******************
+ PBS Known Issues
+******************
+
+-  With PBS Workload Manager, when users try to view a Resource Pool Configuration, the Accelerator
+   field might include an ``unconfigured`` value. This happens when the GPU type information is not
+   set in the PBS ``resources_available`` object.
+
+   Steps to view Resource Pool Configuration:
+
+   -  Select the Cluster tab on the Determined Web UI
+   -  Click on a Resource Pool Card
+   -  Navigate to the Configuration tab on the Resource Pool Details View.
+
+   HPC system administrators can set this information using the following set of commands:
+
+   -  Login as a root level user or use ``sudo -i`` to quickly change to a root level user.
+
+   -  Verify that ``resources_available.accel_type`` is not set for each node we are going to edit.
+
+      -  ``pbsnodes -v <node name> | grep resources_available.accel_type``
+
+   -  Set the desired value for ``resources_available.accel_type`` for the nodes containing GPUs.
+
+      -  ``qmgr -c "set node <node name> resources_available.accel_type=<GPU Type>"``
```

Comment:
Not likely perhaps, but what if more than one accelerator type were installed on a node?

Reply:
Added a step to indicate how set the value in case of multiple GPUs

Reply:
I would drop the 2nd example in favor of properly describing "GPU Type" .     Which is a string describing the GPU model(s) on the node.

Reply:
Added examples for all the steps.